src/zeroIfEmptyString/index.test.js

```diff
@@ -0,0 +1,15 @@
+import zeroIfEmptyString from "./index";
```

Comment:
Someone might assume this returns an actual 0 instead of the string '0' but I think it's fine.